src/SubsBase.API.GraphQL/GraphQL/Types/TrialPeriodSettingsGQLType.cs

```
@@ -0,0 +1,15 @@
+﻿using GraphQL.Types;
+using SubsBase.Data.Models.Json;
+
+namespace SubsBase.API.GraphQL.GraphQL.Types
+{
+    public class TrialPeriodSettingsGQLType : ObjectGraphType<TrialPeriodSettings>
+    {
+        public TrialPeriodSettingsGQLType()
+        {
+            Field<IntervalUnitGQLType>("intervalUnit");
```

why is this set with a string literal not like the others with an expression? this is error prone - probably a silent error that is going to cause a very annoying bug we won't find easily

---

explicit name removed.